components/atoms/Select/single-select.tsx

```diff
@@ -63,7 +77,27 @@ const SingleSelect = ({
           insetLabel && `before:content-[attr(data-inset-label)] before:mr-1 before:font-normal before:text-slate-500`
         )}
       >
-        <p className="grow text-start">{current?.label ?? placeholder}</p>
+        {isTruncated ? (
+          <Tooltip.Provider>
```

Comment:
we have a tooltip component already [here](https://github.com/open-sauced/app/blob/6781c1c010c1f763b5bffdf1b862c30d04f7ee6f/components/atoms/Tooltip/tooltip.tsx#L14) you can use that for consistency

Reply:
I have made this change also could you review it once?

![image](https://github.com/user-attachments/assets/7915ce65-243a-4f95-aea3-916b2a4cace7)